CHANGELOG.md

```diff
@@ -7,12 +7,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [UNRELAESE]
 
+## [1.21.17] - 2024-12-26
+
 ### Fixed
-- Force decimal ‘datatype’ of `numeric` fields  for more accurate display.
+
+- Force decimal `datatype` of `numeric` fields  for more accurate display.
 - Do not destroy the dropdown table/class if it is being used by another container.
 - Fix fields updates with multiple containers via the API.
 
-## [1.21.16] - 2024-11-12
+## [1.21.16] - 2024-16-11
```

Comment:
```suggestion
## [1.21.16] - 2024-11-12
```

Reply:
Sorry but should be '2024-12-12'  (YY-mm-dd)

Reply:
On my calendar, there are only 12 months :)